### PR TITLE
chore(showcase): Add Saobear's blog

### PR DESCRIPTION
**Pull request by PiccoloYu**
I turned my blog into a startup project。

 repo: https://github.com/PiccoloYu/SaoBear-is-Blog
Demo site (in Chinese): https://saobear.online/

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -8946,3 +8946,11 @@
   built_by: n8tb1t
   built_by_url: https://github.com/n8tb1t/
   featured: false
+- title: SaoBear's-Blog
+  main_url: "https://saobear.online/"
```
Comment (laurieontech) [inline docs/sites.yml]:
```suggestion
  main_url: https://saobear.online/
```

File: docs/sites.yml [hunk 2]
```diff
@@ -8946,3 +8946,11 @@
...
+  url: "https://saobear.online/"
```
Comment (laurieontech) [inline docs/sites.yml]:
```suggestion
  url: https://saobear.online/
```

File: docs/sites.yml [hunk 3]
```diff
@@ -8946,3 +8946,11 @@
...
+  source_url: "https://github.com/PiccoloYu/SaoBear-is-Blog"
```
Comment (laurieontech) [inline docs/sites.yml]:
```suggestion
  source_url: https://github.com/PiccoloYu/SaoBear-is-Blog
```

**Comment by PiccoloYu** [comment]
I have corrected the original error and added a description file. Thank

**Comment by laurieontech** [comment]
Thanks for making the changes! Merging now.

**Pull request merged**